play/PlayerRules/CharacterCreation/02_Classes.txt

```diff
@@ -2,645 +2,696 @@ Core classes:
 can be used.
-Passives: Knowledge Nanites - 10; +50 nanites, +10 per level.
+Class Primary Attribute: Intelligence
+Passives: Knowledge Nanites - 10; +50 Nanites, +10 per level.
 Weapon Proficiencies: Pistols.
-Armour Proficiencies: Recon.
-Starting Wealth: $1700+100*Luck - 1400
-Feats: The Sorcorer picks one of the following branches at level 5. Instead of 
+Armor Proficiencies: Recon.
+Starting Wealth: $850 + 100*Luck + 1040
+Feats: The Sorcerer picks one of the following branches at level 5. Instead of 
 gaining more class feats at levels 1-, 15 and 20, they gain access to more 
 powerful spells within that branch. See the Tome of Spells for more 
 information.
-----------------------
-Name: Conjurer
-Conjuration Sorcerers are usually Good aligned. Their research has revealed 
-many powerful spells that often involve creating structures or elemental 
-effects used for support or direct combat.
------------------------
-Name: Phaser
-The Sorcerer who walks the path of the Phaser is one who has realized that 
-Nanites are too powerful to be used by lesser beings. Their knowledge is used 
-to absorb other Nanite user''s attacks, often redirecting them. They can also 
-be powerful controllers.
------------------------
-Name: Consumer
-Consumers are generally the product of a powerful Sorcerer falling into Evil 
-ways. Their Technology favors negating other Nanite users'' abilities, 
-stealing Nanites from others, and denaturing matter. Consumers are powerful 
-direct combatants.
-
-"Gunslinger"
-Passives: [APPLY ONLY TO PISTOL]
-Choose one: Pistoleer lvl 1, Quick draw, Conceal Weapon, Steady Shot, +5 Round 
-capacity to mags, +2 mags to arsenal. Also choose two bonuses and one weakness:
-    - Bonuses: +10 Dam, +10m, -10% Miss Chance, +CS burn, +1 rds in mag;
-    - Weaknesses: -10 Dam, -10m, +10% Miss Chance, -1 magazine;
-Weapon Proficiencies: Pistol.
-Armour Proficiencies: Recon, Light, Concealed.
-Starting Wealth: $1700+100*Luck - 800
+	----------------------
+	Name: Conjurer
+	Conjuration Sorcerers are usually Good aligned. Their research has revealed 
+	many powerful spells that often involve creating structures or elemental 
+	effects used for support or direct combat.
+	-----------------------
+	Name: Phaser
+	The Sorcerer who walks the path of the Phaser is one who has realized that 
+	Nanites are too powerful to be used by lesser beings. Their knowledge is used 
+	to absorb other Nanite user''s attacks, often redirecting them. They can also 
+	be powerful controllers.
+	-----------------------
+	Name: Consumer
+	Consumers are generally the product of a powerful Sorcerer falling into Evil 
+	ways. Their Technology favors negating other Nanite users'' abilities, 
+	stealing Nanites from others, and denaturing matter. Consumers are powerful 
+	direct combatants.
+======================================
+
+
+=====   GUNSLINGER   =====
+[Insert cool tasty text about John Woo]
+Class Primary Attribute: Perception
```

Comment:
I strongly advocate for Dex being primary for gunslingers.

Reply:
How come?

Reply:
Because manual dexterity is more vital to shooting than the ability to see fine details. I can see a target downrange in 1080p, but I can't hit it if I can't hold my weapon steady. Particularly regarding the gunslinger, you don't need superior perception to know to shoot people-shaped targets center of mass (or head), but you do need to be able to point that pistol in the right direction *under massive time pressure* [points at gun-kata]. So, because I see the nature of gunslinger ability usage as revolving around fast-paced gunplay, Dex seems more appropriate.
I do not apply this argument to the marksman because even though they still need to shoot (and that shooting should rely on dex), their ability usage seems flavored more towards deliberate, precise action in the right place at the right moment, with emphasis on picking the right target in the first place, and from much, much farther away.

One of the instructors at my college archery club once related a story about an Olympic archery competitor - who had such poor eyesight that he was legally blind. Because good form (physical stances and motions) is so much more important than anything else when shooting.